news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,75 @@
+
+After peppy said anyone, including new players would be able to get it, they started to believe it can't be performed on a difficulty higher than 4 star, which was completely wrong and resulted in community taking much more time to solve it. That subhint mentioning "Bask in the shade of an ailing mind", referring to basking in the **RGB** shades of an **Insane** diff in particular was completely ignored, because newcomers to osu! would not be able to get this medal.
+
+With a lot of guesses and such an insane 2 hints, 1 being slightly misleading, half of medal hunters entered full Sherlock Holmes mode with the possibilities of the solution, relying more on insanity than logic and drowning the discussion threads with abominations of theories. Some were genuinely good theories that Ephemeral would take note of, and others... well, let's just say Pac-Man theory.
+
+![](/wiki/shared/news/2023-08-28-medal-hunter-titles-second-coming/pacman_theory.jpg)
+
+But, despite all the challenges the community was still determined to find the solution. One of them was [UberFazz](https://osu.ppy.sh/users/8646059), who was going back and forth, seeking for the way to get this medal. At first, they assumed it would be achieving 16,777,216 (256 to the 3rd power) score, because that's how many possible colors there are in RGB. Another guess of theirs was to interact with the osu! website to find a hidden message somewhere on it, which would then lead to a solution. They were also told by one of developers they had insanely close guesses multiple times, but that was just not enough...
+
+The community was trying their best to hunt for the solution, but after an enormous number of attempts that they thought would surely work, insanity started creeping in, and hope was slowly starting to fade. 
+
+But suddenly, after more than 3 weeks, the very last hint was dropped by [Repflez](https://osu.ppy.sh/users/201392) saying that "a message in the last 8 hours was a hint in the right direction", and after reading it, UberFazz found that message and figured exactly what they needed to do. That theory was to set scores with particular combos that relate to the RGB values of a difficulty, which UberFazz guessed was on the "celinsane" difficulty as it literally meant cell + insane just like the medal suggested. Shortly after trying this idea in regards to combo matching each value of the RGB, UberFazz successfully got **Internment** medal and received the "**Medal Hunter**" title, after more than a month since the medal's release!
```

Comment:
```suggestion
But suddenly, after more than 3 weeks, an innocuous bit of encouragement from a watching developer noting that "a message in the last 8 hours was a hint in the right direction" spurned our fated hero into donning his extremely fat thinking cap again. UberFazz read that message and surmised exactly what he needed to do. 

His new theory was to set scores with particular combo counts that relate to the very specific RGB values of a beatmap's difficulty on the website. UberFazz guessed the best difficulty to try was the "celinsane" difficulty as its wording quite literally meant cell + insane, just like the medal suggested. 

After a tense fifteen minutes as UberFazz conjured the requisite three scores into existence, an elated gasp ran through the Medal Hunters server as he found himself the first recipient of the **Internment** medal and the second-ever "**Medal Hunter**" title, after more than a month since the medal's release!
```